x/profile/keeper/msg_server_delete_coordinator_test.go

```diff
@@ -0,0 +1,46 @@
+package keeper
+
+import (
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/spn/testutil/sample"
+	"github.com/tendermint/spn/x/profile/types"
+)
+
+func TestMsgDeleteCoordinator(t *testing.T) {
```

Comment:
I just realized implementing `MsgCreateChain`

For the tests, we must also assert the state changes in the keeper.
If a coordinator is created, we assert that the coordinator and coordinatorByAddress exist in the keeper
If a coordinator is deleted, it doesn't exist anymore in the store, etc....

We can create a new issue since `MsgCreateCoordinator` has been merged, but we can implement those for the current open PRs for messages